software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -24,6 +24,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.google.api.client.repackaged.com.google.common.base.Strings;
```

Comment:
Good idea. I usually spot it when my IDE picks this one, but obviously missed it this time